src/asm/lexer.c

```diff
@@ -921,6 +921,31 @@ void lexer_DumpStringExpansions(void)
 	free(stack);
 }
 
+/* Discards an inline comment */
+static void discardInlineComment(void)
```

Comment:
To be consistent with other languages, I'd rather they be named "block" comments; they can be used inline, but that's not their typical use.

Reply:
will make that change in all places 👍